.github/workflows/push.yml

```diff
@@ -45,13 +45,14 @@ jobs:
                 echo "Checking branch against pattern '$pattern'"
                 if [[ "$GIT_BRANCH" =~ $pattern ]]; then
                   echo "'$GIT_BRANCH' matches '$pattern', continuing with build"
+                  echo "::set-output name=PRE_CHECK_OK::$GITHUB_EVENT_NAME"
                   exit 0
                 fi
             done
             echo "No pattern matches '$GIT_BRANCH', exiting with exit code 1"
             exit 1
           fi
-          echo "::set-output name=PRE_CHECK_OK::true"
+          echo "::set-output name=PRE_CHECK_OK::$GITHUB_EVENT_NAME"
```

Comment:
Not quite sure what's the improvement here? Thought that "true" was clearer if anything. Could not find clear rules about typecasting from string => bool but I guess JS rules apply.

Reply:
Not really an improvement, just some leftover from when I intended to use exit code for flow control.

I felt it was better to have some distinction between output values depending on if it was set by a matching branch or not-create. 

I'm pretty sure we're dealing with js truthy rules, so empty string/null vs any string contents.

Reply:
👍🏻